CoseSignTool/CoseSign1/Usings.cs

```diff
@@ -2,11 +2,15 @@
 // Licensed under the MIT License.
 
 global using System;
+global using System.Collections.Concurrent;

```

Comment:
My bad for moving all the usings into globals.
After some experimentation when converting CoseHandler to netstandard2.0 I realized that not all of the .NET versions under the netstandard2.0 umbrella support global usings :/
Not something I need you to fix now, especially since it's my mistake, but I figured you would want to know.